app/templates/records/record_detail.html

```diff
@@ -101,7 +101,7 @@ <h1 class="tna-hgroup__title">{{ pageTitle }}</h1>
   {{ related_content_block(record) }}
 {% endif %}
 
-{{ related_records_block(record) }}
+{{ related_records_block(record, related_records) }}
```

Comment:
Why is this called a `_block` ? 
Isn't this related records by subjects or series ? Would it be clearer to add `_subjects_or_series`  in macro and file name?